src/meta/src/stream/stream_graph/actor.rs

```diff
@@ -198,37 +198,44 @@ impl ActorBuilder {
                 })
             }
 
-            // "Leaf" node `StreamScan`.
-            NodeBody::StreamCdcScan(stream_scan) => {
+            // "Leaf" node `Filter` used in multi-table cdc backfill plan:
+            // cdc_filter -> backfill -> mview
+            NodeBody::CdcFilter(filter) => {
                 let input = stream_node.get_input();
                 assert_eq!(input.len(), 1);
 
                 let merge_node = &input[0];
                 assert_matches!(merge_node.node_body, Some(NodeBody::Merge(_)));
 
+                let upstream_source_id = filter.upstream_source_id;
```

Comment:
Personally I don't think this is a good name. 😕 In most cases, the upstream is a table or materialized view instead of a CDC source.

Reply:
This is the node_body for `CdcFilter`, I think the name is specific.

Reply:
Ah, you're right.